app/models/concerns/wakeable/has_one.rb

```diff
@@ -19,6 +19,7 @@ def initialize_wakes_graph
     end
 
     def update_wakes_graph
+      return initialize_wakes_graph unless wakes_resource.present?
```

Comment:
@benhutton Again, not in this branch (and I know this wasn't created by this branch), but I'm wondering if we should rename `#initialize_wakes_graph` to `#create_wakes_graph` or something like that because of the connotation in AR and Ruby usually having to do with upon object instantiation in memory and not database creation.